styles.css

```diff
@@ -1 +1,84 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+div {
```

Comment:
Hier kun je beter een class voor gebruiken, als je meerdere divjes in je bestand hebt staan wordt nu alles aangesproken.